packages/hoppscotch-app/src/components/environments/teams/index.vue

```diff
@@ -178,4 +181,14 @@ const getErrorMessage = (err: GQLError<string>) => {
     }
   }
 }
+
+defineActionHandler("modals.team.environment.edit", (args?: any[]) => {
+  const [envName, variableName]: any = args
+  editingVariableName.value = variableName
+  const teamEnvToEdit: TeamEnvironment | undefined =
```

Comment:
Since, the assigned expression already gives info about what is the type of `teamEnvToEdit`. You can drop the explicit type declaration

```suggestion
  const teamEnvToEdit =
```